charts/fluentd/values.yaml

```diff
@@ -53,3 +53,5 @@ s3:
   use_server_side_encryption: ~
 
 priorityClassName: ~
+
+extraEnv: ~
```

Comment:
@danielhoherd is this to be extraEnv or env is enough

Reply:
I like extraEnv because it indicates that we are not defining the env entirely, we are adding to it.